src/Exception/ServiceNotFoundException.php

```diff
@@ -4,15 +4,15 @@
 
 namespace Laminas\ServiceManager\Exception;
 
-use Interop\Container\Exception\NotFoundException;
 use InvalidArgumentException as SplInvalidArgumentException;
+use Psr\Container\NotFoundExceptionInterface;
 
 /**
  * This exception is thrown when the service locator do not manage to find a
  * valid factory to create a service
  */
 class ServiceNotFoundException extends SplInvalidArgumentException implements
     ExceptionInterface,
-    NotFoundException
```

Comment:
This is potentially a BC break for anyone doing `} catch (\ Interop\Container\Exception\ContainerException) {`

Reply:
as it is aliased via `class_alias`, it is treated as the exact same Interface. So actually this should not be a BC break.

Reply:
I've added integration tests to verify that this still works:
https://github.com/laminas/laminas-servicemanager/blob/1758c7e71e7d8e51d82c0e9630b89ce03fca9c68/test/ServiceManagerContainerInteropIntegrationTest.php